cirq/study/sweepable_test.py

```diff
@@ -83,6 +83,18 @@ def test_to_sweeps_iterable_sweeps():
     assert cirq.study.to_sweeps(sweeps) == sweeps
 
 
+def test_to_sweeps_dictionary_of_list():
+    assert cirq.study.to_sweeps({'t': [0, 2, 3]}) == \
```

Comment:
You can use parentheses to avoid the "\"

```
assert (cirq.study.to_sweeps({'t': [0, 2, 3]}) == 
   cirq.study.to_sweeps([{'t': 0}, {'t': 2}, {'t': 3}]))
```